src/components/Navbar.js

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import '../App.css';
+import { MdPerson } from 'react-icons/md';
 
 const Navbar = () => (
   <nav>
-    <h1>Bookstore CMS</h1>
+    <h1 className="header">Bookstore CMS</h1>
```

Comment:
- [ ] This is the styling milestone, I would implore you to follow the styling properties so that your app will follow the design closely. You would notice the font family is specific, kindly add it to all your project and other fonts.

![image](https://github.com/stephenkati/book-store/assets/86200356/365582a5-92df-4570-90ff-a99d8e7923d3)

Reply:
thank you for pointing that out